src/services/completions.ts

```diff
@@ -637,7 +638,7 @@ namespace ts.Completions {
             if (type) {
                 // Filter private properties
                 for (const symbol of type.getApparentProperties()) {
-                    if (typeChecker.isValidPropertyAccess(<PropertyAccessExpression>(node.parent), symbol.name)) {
+                    if (typeChecker.isValidPropertyAccess(<PropertyAccessExpression>(node.parent), symbol.getName())) {

```

Comment:
I was trying to limit where `EscapedString` appears in our public API. (It has to appear since it's a member of `Identifier` and `Symbol`.) Should I not? The downside to swapping `getName` to returning an `EscapedString` is that anyone consuming out services layer will definitely need to add calls to `unescapeIdentifier` to their own code.